frontend/packages/ceph-storage-plugin/src/components/independent-mode/utils.ts

```diff
@@ -30,6 +59,11 @@ export const checkForIndependentSupport = (csv: ClusterServiceVersionKind): bool
   return independent === 'true';
 };
 
+export const getRequiredKeys = (csv: ClusterServiceVersionKind): string[] => {
+  const keys = csv.metadata.annotations?.['external.cluster.ocs.openshift.io/keys'];
+  return keys?.split(',');
```

Comment:
are you expecting keys to be a string?

Reply:
similar to the feature annotation so yes.